wonderitems.py

```diff
@@ -0,0 +1,220 @@
+from Rom import *
```

Comment:
Could the filename be changed to `WonderItems.py` to match the casing of the other main folder files?

Reply:
I just deleted it, it was just a script that I used to find all of the wonderitems. An improved version of this script is including in the potsanity PR.